src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.html

```diff
@@ -43,13 +43,20 @@ <h5><span jhiTranslate="artemisApp.studentExams.workingTime" [ngbTooltip]="getWo
                 :
                 <input formControlName="seconds" type="number" style="width: 60px; height: 35px;" class="text-center" min="0" max="59" step="1" required />
             </div>
-            <button type="submit" class="btn btn-primary" [disabled]="workingTimeForm.invalid || isSavingWorkingTime || examIsVisible()">
+            <button type="submit" class="btn btn-primary mr-2" [disabled]="workingTimeForm.invalid || isSavingWorkingTime || examIsVisible()">
                 <fa-icon [icon]="'save'" [ngbTooltip]="getWorkingTimeToolTip()"></fa-icon>
                 <span jhiTranslate="entity.action.save" [ngbTooltip]="getWorkingTimeToolTip()">Save</span>
             </button>
         </form>
     </div>
 
+    <div class="mb-3">
+        <h5><span jhiTranslate="artemisApp.studentExams.summary">Student Summary</span></h5>
+        <button *ngIf="course?.isAtLeastInstructor" class="btn btn-primary" [routerLink]="'../summary'">
+            <span jhiTranslate="artemisApp.exam.summary">Summary</span>
+        </button>
+    </div>
+
```

Comment:
The button will be displayed next to the save button of the working time edit form. I think this is confusing for the user. I would move it next to the page title.

Reply:
I already changed it so that it will be shown below in a separate section